snuba/subscriptions/executor.py

```diff
@@ -23,8 +23,9 @@ def __init__(self, dataset: Dataset, executor_pool: ThreadPoolExecutor):
         self.__executor_pool = executor_pool
 
     def execute(
-        self, task: ScheduledTask[Subscription], tick: Tick, timer: Timer
+        self, task: ScheduledTask[Subscription], tick: Tick
```

Comment:
I assume none is calling this method right now, right ?

Reply:
Correct.